src/components/catalog-search/catalog-search.controller.ts

```diff
@@ -103,9 +105,9 @@ export class CatalogSearchController implements angular.IController {
     }
   }
 
```

Comment:
Just thinking as I read this, if we aim to avoid `:any` for the type & try to be more specific, should we also try to use `const` as much as possible (instead of `let`), for variables that should not be mutated?  

Nothing to change in this PR, necessarily.

Reply:
+1 to using const when we can. I don't think we take good advantage of ES6 / Typscript. Let me know if there are places you see I should have done that.